src/live.rs

```diff
@@ -32,6 +32,7 @@ use crate::iso9660::{self, IsoFs};
 use crate::miniso;
 
 const INITRD_IGNITION_PATH: &str = "config.ign";
+const INITRD_NETWORK_DIR: &str = "etc/NetworkManager/system-connections";
```

Comment:
Let's change this to `etc/coreos-firstboot-network` as per https://github.com/coreos/fedora-coreos-tracker/issues/841.